flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -241,7 +241,6 @@ under the License.
 			<groupId>org.apache.hive</groupId>
 			<artifactId>hive-exec</artifactId>
 			<version>${hive.version}</version>
-			<scope>provided</scope>
```

Comment:
Please move changes that are not related to the actual issue of adding an e2e test to separate commits.

Reply:
btw I just saw that you marked this PR as `Dependencies (does it add or upgrade a dependency): no` please update this.